src/lib/features/dependent-features/dependent-features-read-model-type.ts

```diff
@@ -1,7 +1,7 @@
 import { IDependency } from '../../types';
 
 export interface IDependentFeaturesReadModel {
-    getChildren(parent: string): Promise<string[]>;
+    getChildren(parents: string[]): Promise<string[]>;
```

Comment:
changing this signature opens options for bulk operations and also works for a single parent